frontend/libs/studio-components/src/components/StudioCodelistEditor/StudioCodeListEditor.tsx

```diff
@@ -27,27 +27,30 @@ import { StudioErrorMessage } from '../StudioErrorMessage';
 export type StudioCodeListEditorProps = {
   codeList: CodeList;
   onChange: (codeList: CodeList) => void;
+  onInvalid?: () => void;
   texts: CodeListEditorTexts;
 };
 
 export function StudioCodeListEditor({
   codeList,
   onChange,
+  onInvalid,
   texts,
 }: StudioCodeListEditorProps): ReactElement {
   return (
     <StudioCodeListEditorContext.Provider value={{ texts }}>
-      <StatefulCodeListEditor codeList={codeList} onChange={onChange} />
+      <StatefulCodeListEditor codeList={codeList} onChange={onChange} onInvalid={onInvalid} />
     </StudioCodeListEditorContext.Provider>
   );
 }
 
-type InternalCodeListEditorProps = Omit<StudioCodeListEditorProps, 'texts'>;
+type InternalStatefulCodeListEditorProps = Omit<StudioCodeListEditorProps, 'texts'>;
```

Comment:
Småpirk: Jeg ville bare kalt denne `StatefulCodeListEditorProps`, siden den bare brukes av `StatefulCodeListEditor`. Bakgrunnen bak `Internal`-prefikset foran `CodeListEditorProps` var for å tydeliggjøre at dette var en generisk type som ble brukt av flere av de interne komponentene (og navnet `CodeListEditorProps` alene brukes jo allerede av rotkomponenten).